openssh/plan.sh

```diff
@@ -11,7 +11,7 @@ pkg_bin_dirs=(bin sbin libexec)
 pkg_include_dirs=(include)
 pkg_lib_dirs=(lib)
 pkg_deps=(core/glibc core/openssl core/zlib)
-pkg_build_deps=(core/coreutils core/gcc core/make)
+pkg_build_deps=(core/coreutils core/gcc core/make core/e2fsprogs)
```

Comment:
Can we break these `pkg_deps` and `pkg_build_deps` onto multiple lines? Its just much nicer 😄